app/services/analytics.rb

```diff
@@ -14,6 +14,12 @@ def track_event(event, attributes = {})
       user_id: attributes[:user_id] || user.uuid,
     }
     ahoy.track(event, analytics_hash.merge!(request_attributes))
+    register_step_from_analytics_event(event, attributes)
+  end
+
+  def register_step_from_analytics_event(event, attributes)
```

Comment:
What do you think about throwing doc auth in the name of this method so it is clear what it is doing?

Reply:
good idea